assets/src/components/libraries/Library.test.jsx

```diff
@@ -47,25 +48,26 @@ describe('Library', () => {
 
   it('shows the search term in the search bar when location has a query parameter', () => {
     history.location.search = '?q=test+search';
-    const { getByPlaceholderText } = render(<Library history={history} slug="bh" />);
-    expect(getByPlaceholderText(/search/i)).toHaveValue('test search');
+    const component = render(<Library history={history} slug="bh" />);
+    expect(component.getByPlaceholderText(/search/i)).toHaveValue('test search');
   });
 
   it('keeps the previous query params in the url even when search is updated', async () => {
     history.location.search = '?toggle=active';
-    const { getByPlaceholderText } = render(<Library history={history} slug="bh" />);
+    const component = render(<Library history={history} slug="bh" />);
 
-    await fireEvent.change(getByPlaceholderText(/search/i), { target: { value: 'test search' } });
+    const searchInput = component.getByPlaceholderText(/search/i);
+    await userEvent.type(searchInput, 'test search');
 
     expect(history.replace).toHaveBeenCalledWith({ search: 'toggle=active&q=test+search' });
     history.location.search = '';
   });
 
   it('removes the search query from the url when search field is cleared', async () => {
     history.location.search = '?q=test+search';
-    const { getByPlaceholderText } = render(<Library history={history} slug="bh" />);
+    const component = render(<Library history={history} slug="bh" />);
 
-    await fireEvent.change(getByPlaceholderText(/search/i), { target: { value: '' } });
+    await userEvent.clear(component.getByPlaceholderText(/search/i));
```

Comment:
It's pretty cool how expressive we can be with the using of the [user-event](https://github.com/testing-library/user-event) library :D